### ISSUE TITLE
importing gnupg fails when zombies are around

### ISSUE THREAD

**Issue by phryk** [isislovecruft/python-gnupg#201]
At least on FreeBSD, `import gnupg` seems to fail as soon as there's any zombie process around.

If I'm reading this traceback right, _find_agent in _meta.py is probably the relevant place for a fix.

```
    import gnupg  
  File "/usr/local/lib/python2.7/site-packages/gnupg/__init__.py", line 23, in <module>
    from . import gnupg
  File "/usr/local/lib/python2.7/site-packages/gnupg/gnupg.py", line 42, in <module>
    from ._meta    import GPGBase  
  File "/usr/local/lib/python2.7/site-packages/gnupg/_meta.py", line 134, in <module>
    class GPGBase(object):  
  File "/usr/local/lib/python2.7/site-packages/gnupg/_meta.py", line 72, in __new__
    if cls._find_agent():  
  File "/usr/local/lib/python2.7/site-packages/gnupg/_meta.py", line 115, in _find_agent
    if (proc.name() == "gpg-agent") and proc.is_running():  
  File "/usr/local/lib/python2.7/site-packages/psutil/__init__.py", line 641, in name
    name = self._proc.name()  
  File "/usr/local/lib/python2.7/site-packages/psutil/_psbsd.py", line 504, in wrapper
    raise ZombieProcess(self.pid, self._name, self._ppid)  
psutil.ZombieProcess: psutil.ZombieProcess process still exists but it's a zombie (pid=39254)
```

If I make a pull request for a fix, should I create a new branch within the repo or use a specified one (!= master)?

**Comment by phryk** [comment]
Bumping because
- it's been over half a year
- it's kind of a critical issue on freebsd
- there's a PR fixing the issue

**Comment by isislovecruft** [comment]
Hi @phryk! Thank you for reporting this, and I'm very sorry it took me so long to get around to addressing it. This _should_ be fixed for you in version pretty-bad-protocol-3.0.1. If not, please feel free to reopen the issue.
